File: Gitesh2808/Java/check-even-odd-no.java
// Date : 22 Jan 2021

// Java program to check whether the number is even or odd



import java.io.*;
class even
{
  public static void main(String args[])
  {
    try 
    {
      DataInputStream x = new DataInputStream(System.in);
      int a;
      System.out.print("\n Enter any number : ");
      a = Integer.parseInt(x.readLine());
      if(a % 2 == 0)
      {
        System.out.print("\n a is even number ");
      }
      else
      {
        System.out.print("\n a is odd number "); 
      }
    }
    catch(Exception e)
    {}
  }
}

File: Durwapatil/C++ Programs/check-even-odd-no.cpp
// Date : 24 Jan 2021

// CPP program to check whether the number is even or odd 



#include <iostream>
using namespace std;

class ab
{
    int a;
    public : int no();
};
int ab :: no()
{
    cout << "Enter any number : \n";
    cin >> a;
    if(a % 2 == 0)
    {
        cout << "Number is even";
    }
    else
    {
        cout << "Number is odd";
    }
}
int main()
{
    ab st;
    st.no();
}



/*
Output : 
    Enter any number : 
    21
    
    Number is odd
*/
